### ISSUE TITLE
Remove Select input SASS styles

### ISSUE THREAD

**Issue by Fcog** [wearenolte/lean-theme#191]
There are some sass files that weren't removed, in atoms/forms

**Comment by nelsonamaya82** [comment]
Yep, I think in general we'll need to clean the theme up from some styles and files that came from the GCC merge.
